src/test/php/web/auth/unittest/OAuth2FlowTest.class.php

```diff
@@ -167,7 +167,7 @@ public function redirects_to_auth_and_passes_scope($scopes) {
   public function redirects_to_auth_when_previous_redirect_incomplete() {
     $fixture= new OAuth2Flow(self::AUTH, self::TOKENS, self::CONSUMER, self::CALLBACK);
     $session= (new ForTesting())->create();
-    $session->register('oauth2::flow', ['state' => 'PREVIOUS_STATE', 'target' => self::SERVICE]);
+    $session->register('oauth2::flow', ['target' => ['PREVIOUS_STATE' => self::SERVICE]]);
 
```

Comment:
This illustrates perfectly why we should use a different key other than *target* here: If the new implementation encounters an old session, accessing the value via `$value['target']`, it will get a string, and not the array it expects!